### PR TITLE
Remove div class

### LINKED ISSUE #2012: MNE report broken, open tags in final HTML
@teonlamont git praise tells made the you made the latest changes on report.py ;) Our latest reports are broken, images goe into the contents ...

### PR DESCRIPTION
**Pull request by teonlamont**
closes #2012.

### CODE REVIEW

**Comment by Eric89GXL** [comment]
BTW how did this one get past Travis? Was it not tested? Or is it that it just bungled the rendering, so it could only be found via visual inspection?

**Comment by agramfort** [comment]
@dengemann merge if happy

**Comment by teonlamont** [comment]
@Eric89GXL it was in the rendering, I think it would have only been found by inspection

**Comment by teonlamont** [comment]
@Eric89GXL with the closes #XXX, does it close after the pr is done or when it's mentioned?

**Comment by Eric89GXL** [comment]
On merge. As soon as one of us clicks the button it happens automatically.

**Comment by dengemann** [comment]
I cannot try now. Later. But it's probably the culprit.

**Comment by dengemann** [comment]
runing test now.

File: mne/report.py [hunk 1]
```diff
@@ -553,7 +553,6 @@ def _build_html_slider(slices_range, slides_klass, slider_id):
     {{endif}}
     {{if comments is not None}}
         <br><br>
-        <div class="comments">
```
Comment (mainakjas) [inline mne/report.py]: something looks suspicious about this. Why don't your remove the closing `</div>` ?

Comment (teonlamont) [inline mne/report.py]: doesn't `<div style>` need a `</div>`

Comment (Eric89GXL) [inline mne/report.py]: I think the closing div was missing, which was the problem?

Comment (Eric89GXL) [inline mne/report.py]: ...or equivalently the opening one didn't have a closing one, which makes removing the opening one a solution?

Comment (teonlamont) [inline mne/report.py]: @Eric89GXL exactly. i had two openers and only one closer

**Comment by dengemann** [comment]
negative, no fix.

**Comment by dengemann** [comment]
screenshot 2015-04-23 19 31 52

You see the images are now behind the contents.

**Comment by teonlamont** [comment]
and that didn't happen before?

**Comment by dengemann** [comment]
> and that didn't happen before?
 
nope.

File: mne/report.py (hunk 1 above)
Comment (mainakjas) [inline mne/report.py]: @teonlamont : I see a matching `<div>` to the one you removed. Indentation is critical to identifying the matching divs.

**Comment by teonlamont** [comment]
a separate issue but related to the report: I just realized that with this current implementation `comments` will be repeated if a list of figs/images are supplied. the behavior for list to consider each item to be a separate line. we could remove this and make it so that each item corresponds to the respective figs/images

**Comment by teonlamont** [comment]
I can't find where this is going wrong syntactically...

**Comment by dengemann** [comment]
Teon, trying your latest commit.

**Comment by dengemann** [comment]
Thanks Teon !! All is fine again :)

**Pull request merged**